packages/h2/h2.0.7.0/opam

```diff
@@ -15,7 +15,7 @@ depends: [
   "alcotest" {with-test}
   "yojson" {with-test}
   "hex" {with-test}
-  "base64"
+  "base64" {>= "3.0.0"}
```

Comment:
could you send a separate PR for those new constrains with the error that you encountered?

Reply:
Did this in https://github.com/ocaml/opam-repository/pull/21963